src/components/Hero.tsx

```diff
@@ -1,17 +1,41 @@
-import { useState } from "react";
+import { useState, useRef } from "react";
 import searchNormal from "../assets/searchNormal.svg";
 import starIcon from "../assets/starIcon.svg";
 import issueIcon from "../assets/issueIcon.svg";
+import cmdKIcon from "../assets/cmdK.svg";
 import { fetchRecommendations } from "../lib/supabase";
 import humanizeNumber from "../lib/humanizeNumber";
-import { useDebounce, useDidUpdate } from "rooks";
+import { useDebounce, useDidUpdate, useKeys } from "rooks";
 
 const Hero = () => {
+  const containerRef = useRef<Document>(document);
+  const searchBoxRef = useRef<HTMLInputElement>(null);
   const [searchTerm, setSearchTerm] = useState("");
   const setValueDebounced = useDebounce(setSearchTerm, 500);
   const [fetchedData, setFetchedData] = useState<DbRecomendation[]>([]);
   const [hasFocus, setFocus] = useState(false);
 
+  useKeys(
+    ["ControlLeft", "KeyK"],
```

Comment:
This works with "Control" but the UI shows "command." I could not find in the docs, but can we include multiple keys? 

The correct key should be "MetaLeft" and not "ControlLeft" - https://www.toptal.com/developers/keycode/for/meta

With that said, I think this should work for windows users.

Reply:
I dont think it's possible to use multiple keys, I will have to call two useKeys.

Reply:
We can actually use [useKeyBindings](https://react-hooks.org/docs/useKeyBindings) to bind the 2 pairs to the same callback. I will experiment with this a bit later and check if I can provide some better code, however, feel free to play with the hook, it should work as intended! 🍕

Reply:
Cool, I just updated the PR with possible fix (using two useKeys)

Reply:
It works